docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -8,360 +8,234 @@ ms.date: 06/12/2023
 
 # Prepare .NET libraries for trimming
 
-The .NET SDK makes it possible to reduce the size of self-contained apps by [trimming](trim-self-contained.md), which removes unused code from the app and its dependencies. Not all code is compatible with trimming, so .NET 6 provides trim analysis warnings to detect patterns that may break trimmed apps. To resolve warnings originating from the app code, see [resolving trim warnings](#resolve-trim-warnings). This article describes how to prepare libraries for trimming with the aid of these warnings, including recommendations for fixing some common cases.
+The .NET SDK makes it possible to reduce the size of self-contained apps by [trimming](trim-self-contained.md). Trimming removes unused code from the app and its dependencies. Not all code is compatible with trimming. .NET provides trim analysis warnings to detect patterns that may break trimmed apps. This article:
 
-## Enable library trim warnings
+* Describes how to prepare libraries for trimming.
+* Provides recommendations for [resolving common trimming warnings](#resolve-trim-warnings).
+
+## Prerequisites
+
+[.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet) or later.
 
-> [!TIP]
-> Ensure you're using the .NET 6 SDK or later for these steps. They will not work correctly in previous versions.
+## Enable library trim warnings
 
-There are two ways to find trim warnings in your library:
+Trim warnings in a library can be found with either of the following methods:
 
-  1. Enable project-specific trimming using the `IsTrimmable` property.
-  2. Add your library as a reference to a sample app, and trim the sample app.
+* Enabling project-specific trimming using the `IsTrimmable` property.
+* Creating a trimming test app that uses all the APIs in the library, and enabling trimming for the test app.
```

Comment:
This is not precise - while it's technically true that the test app will "reference" all of the APIs of the library, it is not done manually - the sentence here sounds like the developer will have to write an app which exercises all APIs - which sounds very tedious.